src/storage/src/controller/rehydration.rs

```diff
@@ -183,6 +185,39 @@ where
                 .await;
         }
 
+        for export in self.exports.values_mut() {
```

Comment:
The rehydration logic is also used when the controller restarts? I'm asking because we removed the logic from there.

Reply:
it is, that's correct!